volatility3/cli/__init__.py

```diff
@@ -397,6 +414,41 @@ def run(self):
                 )
                 ctx.config[address] = value
 
+        if args.amazon_bucket:
```

Comment:
Yep, please don't tinker with this.  It will only add support to the CLI and require any other clients to know about and implement similar options, whereas the URL option will work across everything if the handler is installed.